share/completions/magento.fish

```diff
@@ -0,0 +1,720 @@
+###
+# executes fish MySQL commands
+#
+# To make this work, you'll need to create a __fish user in your
+# database, using the following mysql commands:
+#
+#   mysql> CREATE USER '__fish'@'localhost' IDENTIFIED BY 'your random pw';
+#   mysql> GRANT SELECT, SHOW ON * . * TO '__fish'@'localhost';
+#   mysql> FLUSH PRIVILEGES;
+#
+# If no credentials are given, it will simply print nothing, so further commands
+# won't show suggestions
+###
+function __fish_print_magento_mysql_command -d "Executes a MySQL command using the fish user"
+
+### IMPORTANT: ENTER FISH CREDENTIALS HERE #####
+  set -l fish_mysql_user __fish
+  set -l fish_mysql_pass fish_12479284903518
+################################################
+
+  set -l result (mysql -u{$fish_mysql_user} -p{$fish_mysql_pass} -e $argv --skip-column-names 2>&1 | grep -v "mysql:")
+  if [ (echo $result | grep ERROR) ]
```

Comment:
As a general comment (this will be remove later):

- We prefer `string match` for this since it doesn't require forking off an external process

- The test is unnecessary since both grep and `string match` have a "-q" option that silences output and returns 0 if there are matches, 1 if there are none.

So this would be `if string match -qr 'ERROR' -- $result` or `if echo $result | grep -q ERROR`.